pop3/commands/uidl/uidl.go

```diff
@@ -25,10 +25,10 @@ func Process(c *Client, cmd Command) Reply {
 		}
 	}
 	envs := db.List(c.GetString("authenticated_as"))
-	if cmd.Args == "" {
-		msg := []string{"\r\n"}
+	if cmd.Args == "" { // clawsmail says this an invalid response
+		msg := []string{"unique-id listing follows"}
 		for i, env := range envs {
-			msg = append(msg, fmt.Sprintf("%v %v", i, env.ID))
+			msg = append(msg, fmt.Sprintf("%v %v", i, env.ID)) // shouldnt we count from 1 ?
```

Comment:
Should we? It doesn't seem I did that in `gomaild`:
https://github.com/trapped/gomaild/blob/master/processors/pop3/cmdprocessor/uidl/uidl.go
https://github.com/trapped/gomaild/blob/master/processors/pop3/cmdprocessor/list/list.go